src/reactviews/pages/QueryResult/table/plugins/headerFilter.plugin.ts

```diff
@@ -317,7 +338,7 @@ export class HeaderFilter<T extends Slick.SlickData> {
                 }
                 this.setButtonImage(
                     $menuButton,
-                    this.columnDef.filterValues.length > 0,
+                    this.columnDef.filterValues!.length > 0,
```

Comment:
What happens if this is undefined?  Maybe use `if (fitlerValues && filterValues.length > 0)`